pages/linux/kdocker.md

```diff
@@ -0,0 +1,28 @@
+# kdocker
+
+> Easily dock applications to system tray.
+> More information: <https://github.com/user-none/KDocker>.
+
+- Click on a window and send it to system tray:
```

Comment:
This description is strange, maybe you can use something similar to https://github.com/tldr-pages/tldr/blob/main/pages/common/xkill.md

Reply:
Would you like to suggest?
If not I'll happily do it :)

Reply:
```suggestion
- Display a cursor to send a window to system tray when pressing the left mouse button (press any other mouse button to cancel):
```